python/ray/experimental/sgd/tf/tf_trainer.py

```diff
@@ -92,24 +95,65 @@ def __init__(self,
                 for i, worker in enumerate(self.workers)
             ])
 
-    def train(self):
+        import tensorflow as tf
+        self._Progbar = tf.keras.utils.Progbar
```

Comment:
do we need self._Progbar? would it be possible to import in _generic_model_driver